statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -923,7 +923,7 @@ def is_stable(self, verbose=False):
         """
         return is_stable(self.coefs, verbose=verbose)
 
-    def simulate_var(self, steps=None, offset=None, seed=None):
+    def simulate_var(self, steps=None, offset=None, initial_state=None, seed=None):
```

Comment:
Keywords should match.  I would go with `initial_values` which is clearest.  This keyword must be added at the end since it is new and putting it in the middle will break any position only use. This should be keyword only

```
def simulate_var(self, steps=None, offset=None, seed=None, *, initial_values=None):
```